docs/src/pages/demos/chips/ChipsPlayground.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { withStyles } from '@material-ui/core/styles';
-import MarkdownElement from 'docs/src/modules/components/MarkdownElement';
+import MarkdownElement from '../../../modules/components/MarkdownElement';
```

Comment:
The demo details are disabled. This TypeScript conversion won't be seen by the user's.